drsm/updates.go

```diff
@@ -240,6 +241,7 @@ func (d *Drsm) addChunk(full *FullStream) {
 		pod = d.addPod(full)
 	}
 	did := full.Id
+	log.Printf("received Chunk Doc: %v", full)
```

Comment:
Review all logs if they are really useful and how frequently they come in the pod output